qutip/qobjevo.py

```diff
@@ -217,21 +230,44 @@ def __getitem__(self, i):
 
 
 class QobjEvo:
-    """A class for representing time-dependent quantum objects,
-    such as quantum operators and states.
-
-    The QobjEvo class is a representation of time-dependent Qutip quantum
-    objects (Qobj). This class implements math operations :
-        +,- : QobjEvo, Qobj
-        * : Qobj, C-number
-        / : C-number
-    and some common linear operator/state operations. The QobjEvo
-    are constructed from a nested list of Qobj with their time-dependent
-    coefficients. The time-dependent coefficients are either a funciton, a
-    string or a numpy array.
-
-    For function format, the function signature must be f(t, args).
-    *Examples*
+    """
+    A class for representing time-dependent quantum objects, such as quantum
+    operators and states.
+
+    Basic math operations are defined:
+
+    - ``+``, ``-`` : :class:`~QobjEvo`, :class:`~Qobj`, scalars.
+    - ``*``: :class:`~Qobj`, C number
+    - ``/`` : C number
+
+    This object is constructed by passing a list of :obj:`~Qobj` instances,
+    each of which *may* have an associated scalar time dependence.  The list is
+    summed to produce the final result.  In other words, if an instance of this
+    class is :math:`Q(t)`, then it is constructed from a set of constant
+    :obj:`~Qobj` :math:`\\{Q_k\\}` and time-dependent scalars :math:`f_k(t)` by
+
+    .. math::
+
+        Q(t) = \\sum_k f_k(t) Q_k
+
+    If a scalar :math:`f_k(t)` is not passed with a given :obj:`~Qobj`, then
+    that term is assumed to be constant.  The next section contains more detail
+    on the allowed forms of the constants, and gives several examples for how
+    to build instances of this class.
+
+    **Time-dependence formats**
+
+    There are three major formats for specifying a time-dependent scalar:
+
+    - Python function
+    - string
+    - array
```

Comment:
@hodgestar about the `QobjEvo` documentation - added a paragraph or two explaining the bit that went missing.